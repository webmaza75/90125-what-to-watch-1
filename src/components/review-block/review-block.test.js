import React from 'react';
import renderer from 'react-test-renderer';

import ReviewBlock from './review-block.jsx';
import reviews from '../../mocks/reviews.js';

it(`ReviewBlock correctly renders`, () => {
  const tree = renderer
    .create(<ReviewBlock
      list={reviews}
      idx={`left`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
