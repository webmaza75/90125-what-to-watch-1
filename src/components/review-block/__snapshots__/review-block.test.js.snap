// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewBlock correctly renders 1`] = `
<div
  className="movie-card__reviews-col"
>
  <div
    className="review"
  >
    <blockquote
      className="review__quote"
    >
      <p
        className="review__text"
      >
        Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
      </p>
      <footer
        className="review__details"
      >
        <cite
          className="review__author"
        >
          Kate
        </cite>
        <time
          className="review__date"
          dateTime="2019-05-08"
        >
          May 8, 2019
        </time>
      </footer>
    </blockquote>
    <div
      className="review__rating"
    >
      8,9
    </div>
  </div>
  <div
    className="review"
  >
    <blockquote
      className="review__quote"
    >
      <p
        className="review__text"
      >
        Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
      </p>
      <footer
        className="review__details"
      >
        <cite
          className="review__author"
        >
          Kate Muir
        </cite>
        <time
          className="review__date"
          dateTime="2019-05-08"
        >
          May 8, 2019
        </time>
      </footer>
    </blockquote>
    <div
      className="review__rating"
    >
      9,0
    </div>
  </div>
</div>
`;
